### DISCUSSION TITLE
Improvements to package comparison table?

### DISCUSSION THREAD

**Discussion by farleykr** [djangopackages/djangopackages#665] [Grids]
Hello,  I've been a user of djangopackages.com for a long time, but have recently found myself using it a lot because I'm now building/maintaining a django app at work.  I've always found the table that compares different packages very difficult to read.  Would anyone be open to a PR that includes a re-organization of this table?  I understand this would not be a small change which is why I wanted to open an issue to ask this question.  I apologize if opening an issue is not the best way to handle this request.  Thank you for your time.

**Comment by rexhepberlajolli** [comment]
@farleykr , We decided that we will start to convert project to SPA, and we will do a better implementation of comparison table you are welcome to contribute on that.

**Comment by fabiocaccamo** [comment]
Any news about the status of the development?

**Comment by farleykr** [comment]
I apologize for the delay.  I think I misunderstood @rexhepberlajolli 's comment about the SPA.  I was under the impression that a baseline amount of work would need to be done on the SPA before the package comparison table was ready for changes.
